src/main/resources/application.yml

```diff
@@ -22,6 +22,18 @@ spring:
     password: ${MYSQL_PASSWORD:password}
     driver-class-name: org.drizzle.jdbc.DrizzleDriver
 
+  mail:
```

Comment:
Can you attach a demo video with Gmail settings to show that the email is sending successfully with Gmail?